### PR TITLE
Add common dimensions across compliance queries and migrate queries to service.sp files. Closes #39

### LINKED ISSUE #39: Add common dimensions across compliance queries and migrate queries to service.sp files
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by madhushreeray30**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Review by Priyanka-Chatterjee-2000 — CHANGES_REQUESTED** [review]
Please can you take a look at the suggested changes. Thanks!!

**Review by Priyanka-Chatterjee-2000 — CHANGES_REQUESTED** [review]
Please can you take a look at the suggested changes. Thanks!!

**Comment by Priyanka-Chatterjee-2000** [comment]
LGTM

**Review by Priyanka-Chatterjee-2000 — APPROVED** [review]

File: query/group.sp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        when visibility = 'Public' then 'alarm'
+        else 'ok'
+      end status,
+      case
+        when visibility = 'Public' then title || ' is public.'
+        else title || ' is not public.'
+      end reason
+      ${local.common_dimensions_sql}
+    from
+      azuread_group;
```
Comment (khushboo9024) [inline query/group.sp:15]: @Priyanka-Chatterjee-2000  `azuread_group` table support tags any reason behind not adding tag dimenstion?

**Comment by misraved** [comment]
@madhushreeray30 could you please update the `docs/index.md` file with the common dimension examples?

**Pull request merged**